### PR TITLE
Fixed cursor color issue

### PR DESCRIPTION
**Pull request by askNilesh**
BEFORE: 
image

AFTER
image

image

### CODE REVIEW

File: tarka-ui/src/main/java/com/tarkalabs/tarkaui/components/TUISearchBar.kt [hunk 1]
```diff
@@ -85,7 +86,8 @@ import kotlinx.coroutines.delay
     interactionSource = interactionSource,
     textStyle = TUITheme.typography.body6.copy(color = TUITheme.colors.inputText),
     keyboardActions = keyboardAction,
-    keyboardOptions = keyboardOption
+    keyboardOptions = keyboardOption,
+    cursorBrush = SolidColor(TUITheme.colors.inputText)
```
Comment (rajajawahar) [inline tarka-ui/src/main/java/com/tarkalabs/tarkaui/components/TUISearchBar.kt:90]: Dude, what is this property for ?

**Review by sheshadhreens — APPROVED** [review]

File: tarka-ui/src/main/java/com/tarkalabs/tarkaui/components/TUISearchBar.kt (hunk 1 above)
Comment (askNilesh) [reply tarka-ui/src/main/java/com/tarkalabs/tarkaui/components/TUISearchBar.kt:90]: it is for cursor color @rajajawahar

**Review by Younes-Charfaoui — APPROVED** [review]
@raja shall we move these modules to our technician app?

File: tarka-ui/src/main/java/com/tarkalabs/tarkaui/components/TUISearchBar.kt [hunk 2]
```diff
@@ -98,7 +100,7 @@ import kotlinx.coroutines.delay
         top = 0.dp, bottom = 0.dp, end = 10.dp, start = 10.dp
       ),
       colors = TextFieldDefaults.colors(
-        cursorColor = TUITheme.colors.inputText,
+        cursorColor = TUITheme.colors.error,
```
[the remark this answers is not in the document]
Comment (askNilesh) [reply tarka-ui/src/main/java/com/tarkalabs/tarkaui/components/TUISearchBar.kt:103]: This `cursorColor` is not working for `BasicTextField` to change  color for textfield cursor color @Younes-Charfaoui

**Review by deekshitheam360 — APPROVED** [review]

**Pull request merged**